SKSpriteButton/Classes/SKSpriteButton.swift

```diff
@@ -47,197 +53,137 @@ public class SKSpriteButton: SKSpriteNode {
         case releaseFast
         case reentry
     }
-    
-    /// Read-only property for checking whether the button is in `disabled`
-    /// status.
-    public var disabled: Bool {
-        switch status {
-        case .disabled: return true
-        default: return false
+
+    /// Button becomes a toggle switch that switches between normal and toggledOn. When true, button will not
+    /// invoke the touchesEnded event listener but rather the touchesToggleOn/Off listener
+    public var isToggleMode: Bool = false
```

Comment:
As we discussed before, this is really encapsulating something that should be represented by a new class, it can get unscalable when there are different type buttons added.